### PR TITLE
[#4910] Add examples to NestedTernaryOperator and OneLineConditional

### PR DESCRIPTION
**Pull request by unkmas**
Added examples.

Changes are small and belongs to one topic, so i decided to put them in one PR

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [ ] Added tests.
* [ ] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Run `rake default` or `rake parallel`. It executes all tests and RuboCop for itself, and generates the documentation.

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: lib/rubocop/cop/style/one_line_conditional.rb [hunk 1]
```diff
@@ -5,6 +5,15 @@ module Cop
...
       # Checks for uses of if/then/else/end on a single line.
+      #
+      # @example
+      #   # bad
+      #   if foo then boo else doo end
+      #   if foo then boo end
+      #   unless foo then boo end
+      #
+      #   # good
+      #   foo ? boo : doo
```
Comment (bbatsov) [inline lib/rubocop/cop/style/one_line_conditional.rb]: I think here you should also mention that the multi-line versions of those expressions would be fine. Or the suffix forms for the examples without else.

Comment (bbatsov) [reply lib/rubocop/cop/style/one_line_conditional.rb]: Actually I think the cop doesn't even check expressions without an `else`.

Comment (unkmas) [reply lib/rubocop/cop/style/one_line_conditional.rb]: @bbatsov you are right >_<. I'm so used to not write oneliners with "if/then/end" style, that I missed that examples in specs :(

Fixed.

**Comment by bbatsov** [comment]
👍

**Pull request merged**